content/docs/gke/customizing-gke.md

```diff
@@ -200,6 +200,24 @@ kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/container
 Set [`enable_tpu:true`](https://github.com/kubeflow/manifests/blob/4d2939d6c1a5fd862610382fde130cad33bfef75/gcp/deployment_manager_configs/cluster-kubeflow.yaml#L80)
 in `${KF_DIR}/gcp_config/cluster-kubeflow.yaml`.
 
+
+### Specify a minimum CPU
+
+Certain instructions sets or hardware features are only available on specific CPUs, so to ensure your cluster utilizes the appropriate hardware, you need to set a minimum CPU value. 
+
+In brief, inside gcp_config/cluster.jinja change the 'minCpuPlatform' property for the CPU node pool. For example, 'Intel Broadwell' becomes 'Intel Skylake'. Setting a minimum CPU needs to occur during cluster/node creation; it cannot be applied to an existing cluster/node.
+
+More detailed instructions follow.
+
+* Choose a zone you want to deploy in that has your required CPU. Zones are listed [here](https://cloud.google.com/compute/docs/regions-zones/).
+
+* Deploy Kubeflow normally as specified [here](https://www.kubeflow.org/docs/gke/deploy/deploy-cli/) but stop at section ["Set up and deploy Kubeflow"](https://www.kubeflow.org/docs/gke/deploy/deploy-cli/#set-up-and-deploy-kubeflow). Instead, navigate to section ["Alternatively, set up your configuration for later deployment"](https://www.kubeflow.org/docs/gke/deploy/deploy-cli/#alternatively-set-up-your-configuration-for-later-deployment). Then follow step 1.
```

Comment:
We use relative paths for all links that point to a page within the Kubeflow doc site. This ensures that reviewers can remain within a preview site without inadvertently jumping to the production site. It also means that the links will work as expected when we eventually start offering the docs in multiple languages.

The relative links must start at the path element `/docs`.

So, in the first case, the link should look like this:

```
/docs/gke/deploy/deploy-cli/
```